codegen/build_dbc.py

```diff
@@ -76,7 +76,7 @@ def get_key_by_val(dictionary, val):
         source = get_key_by_val(device_enum, can_frame.source)
         # Checks for critical messages to make sure an ACK is added later
         if can_frame.is_critical != None and can_frame.is_critical:
-            ACKABLE_MESSAGES.add(str(can_frame.source))
+            ACKABLE_MESSAGES[msg_id] = ((can_frame.target).replace(" ", "")).split(",")
```

Comment:
this can just be `can_frame.target.split(", ")` I think

Reply:
I added the replace since sometimes people leave spaces in between each receiver 
ie `target "BMS_CARRIER, SOLAR, MOTORCONTROLLER`
notice the small spaces, the replace() removes these spaces which makes sure when we are parsing it later and comparing it with the enum (with all the names the senders/receivers) that it will not throw an error.

Reply:
Hmm, that works too, though I added it in case there were some extra spaces within the string itself. Just to make it more foolproof for whoever is editing the asciipb file. Though the format is technicall like
`target "BMS_CARRIER, SOLAR, MOTORCONTROLLER` putting a space in the wrong place will cause issues if I don't remove them.

Reply:
I’d rather enforce a schema than allow various combinations of spaces and commas. If you’re concerned about extra spaces in the tokens we can add a check during iteration to ensure it’s a valid target.

Reply:
I’ll leave that up to you though, I guess it’s just style